tests/testthat/test_blockForest_surv_blockforest.R

```diff
@@ -0,0 +1,15 @@
+test_that("autotest", {
+  set.seed(1)
```

Comment:
also use `with_seed` or `local_seed()
```suggestion
  withr::local_seed(1)
```